### PR TITLE
Updated .env for GLogin setup in React Setup Docs

### PR DESCRIPTION
**Pull request by H-SM**
hey @AmanNegi , I finished up updating the doc for GLogin Setup in the React Application. Hope it meet the expectations and gets merged up soon. 

Issue in hand:  `[ doc ] : Update .env setup in React Setup Docs` #153 

Ps.: don't forget to mark the PR as `hacktoberfest-accepted`. Have a great day.

### CODE REVIEW

**Review by AmanNegi — CHANGES_REQUESTED** [review]
Here's some suggestions.

File: frontend/react/README.md [hunk 1]
```diff
@@ -70,4 +70,57 @@ npm run dev
...
+    
+3. Open the `.env` file in a code editor.
+
 > Create your own credientials, and add the appropriate environment variables.
+    
+4. Add the following lines to your `.env` file, replacing `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` with the client ID and client secret you obtained from the Google Developer Console:
+    
+```
+VITE_GLOGIN_CLIENT_ID="YOUR_CLIENT_ID"
+VITE_GLOGIN_CLIENT_SECRET="YOUR_CLIENT_SECRET"
```
[the remark this answers is not in the document]
Comment (H-SM) [reply frontend/react/README.md:121]: On it..

[the remark this answers is not in the document]
Comment (H-SM) [reply frontend/react/README.md:121]: hey @AmanNegi , I looked into these further today... These are the api config for the `firebase` connection for the application, and not for the `google oauth`... oauth requires only the secret and the client ID to the web client thats it... 
Hope you look into it again... :)

[the remark this answers is not in the document]
Comment (AmanNegi) [reply frontend/react/README.md:121]: You are right @H-SM, upon researching I found that we are only using one `.env` variable for Google Auth, which is:
```env
VITE_GOOGLE_CLIENT_ID=""
```
So just adding this should be fine.

[the remark this answers is not in the document]
Comment (H-SM) [reply frontend/react/README.md:121]: on it....

**Comment by H-SM** [comment]
did the required changes here... hope it gtg now. 👌

Ps. @AmanNegi can I contact you personally by any means? I wish to talk to you regrading a few things, I saw your profile and would love to clear a few doubts of mine... that would be great.. Thanks.

**Review by AmanNegi — APPROVED** [review]

**Comment by AmanNegi** [comment]
@H-SM LGTM! Feel free to mail me at asterjoules@gmail.com or feel free to DM me on Discord (My Username: asterjoules).

**Pull request merged**